src/app/components/my-stats/my-stats.component.html

```diff
@@ -0,0 +1,75 @@
+<tui-island class="course-island" size="m">
+    <div style="width:68%; float:left;">
```

Comment:
Never use inline styles.
In case you want to use styles you have to do in in the scss file.
For this case you have to use taiga ui grids and not styles.
https://taiga-ui.dev/grid